client/components/main/About.jsx

```diff
@@ -7,32 +7,33 @@ import {
 import empowerLaLogo from '@assets/empower_la_logo.png';
 import hackForLaLogo from '@assets/hack_for_la_logo.png';
 import codeForAmericaLogo from '@assets/code_for_america_logo.png';
+import mobileAppIcon from '@assets/mobile_app_icon.png';
+import databaseIcon from '@assets/database_icon.png';
+import publishIcon from '@assets/publish_icon.png';
+import upliftIcon from '@assets/uplift_icon.png';
+import visualizeIcon from '@assets/visualize_icon.png';
 
 const useStyles = makeStyles({
   root: {
     color: 'black',
     backgroundColor: 'white',
     alignItems: 'center',
     justifyContent: 'center',
-    padding: '2em',
+    padding: '2rem',
     '& h1': {
-      fontSize: '2.5em',
+      fontSize: '2.5rem',
     },
     '& h2': {
-      fontSize: '2.25em',
+      fontSize: '2.25rem',
     },
     '& img': {
       maxWidth: '100%',
-      height: 'auto',
-      display: 'inline-block',
+      display: 'block',
+      margin: '0 auto',
     },
   },
-  gridItem: {
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
-    paddingTop: '4em',
-    paddingBottom: '6em',
+  gridContain: {
```

Comment:
since all instances of `className={classes.gridContain}` can be replaced with `justify="space-between"` component property, we can remove the definition of `gridContain`

Reply:
Will delete.